app/views/pages/_contributor_blurb.html.erb

```diff
@@ -0,0 +1,9 @@
+<div class="gridManyItem center">
+  <%= react_component('Modal', props: {
+    element: { component: 'Avatar', props: { src: ProfilePicture.normalize_url(blurb["image"]), name: blurb['name'], alt: "Contributor: #{blurb['name']}"} },
+    title: nil,
+    body: blurb["profile"]
```

Comment:
We still want to have:
```
<i class="fa fa-location-arrow fa-inline"></i><%= " #{blurb["location"]}" %>
        <br /><%= content_tag(:i, '', class: "#{get_icon_class(blurb["social"])} fa-inline") %><%= link_to(blurb["link_name"], blurb["link"], target: 'blank') %>
```

You can pass a partial in this case and it won't cause problems cause there's no nested React component haha.

Reply:
Ah, I will cover this all off. Still a WIP :D